nexus/src/app/background/inventory_collection.rs

```diff
@@ -153,64 +151,14 @@ async fn inventory_activate(
     Ok(collection)
 }
 
-/// Determine which sleds to inventory based on what's in the database
```

Comment:
I'm probably going to undo the commit containing this change (moving the sled querying logic to common). Initially I thought that I should potentially query the sled agents directly, but it seems that using the records stored by this background job is sufficient even though it adds some latency.